Missing initializer for member ‘sigaction::sa_mask’

Hi Bill,

With WinFsp 2020 / 1.6, with FUSE Cygwin API, I get the following warning :
```
/usr/include/fuse/winfsp_fuse.h: In function ‘int fsp_fuse_set_signal_handlers(void*)’:
/usr/include/fuse/winfsp_fuse.h:363:41: warning: missing initializer for member ‘sigaction::sa_mask’ [-Wmissing-field-initializers]
  363 |     struct sigaction oldsa, newsa = { 0 };
      |        
```

Could it be possible to mute it ?

Many thanks 👍 

Ben

--- a/inc/fuse/winfsp_fuse.h
+++ b/inc/fuse/winfsp_fuse.h
@@ -360,7 +360,10 @@ static inline int fsp_fuse_set_signal_handlers(void *se)
 
     static sigset_t sigmask;
     static pthread_t sigthr;
-    struct sigaction oldsa, newsa = { 0 };
+    struct sigaction oldsa, newsa;
+
+    # memset instead of ={0} to avoid a GCC -Wmissing-field-initializers warning
+    memset(&newsa, 0, sizeof newsa);
 
     if (0 != se)
     {

